src/ProcessManager/FirefoxManager.php

```diff
@@ -118,7 +118,9 @@ private function getDefaultOptions(): array
             'host' => '127.0.0.1',
             'port' => 4444,
             'path' => '/status',
-            'capabilities' => [],
+            'capabilities' => [
+                'acceptInsecureCerts' => true,
```

Comment:
Required on firefix headless because the "Symfony dev CA" isn't recognized there :shrug:

Reply:
OK, removed. The rest is ready.